src/clarity-angular/emphasis/alert/alerts-pager.html

```diff
@@ -0,0 +1,15 @@
+<div class="pager-control">
+    <div class="pager-item">
```

Comment:
I don't really need the class "pager-item", turns out I didn't do anything with it.  Keeping the button in a div though does constrain the focus outline after clicking on page-up/page-down nicely around the button.  Without the div the focus outline is quite large and does not look nice.  Are you OK with simply replacing the "pager-item" class with more meaningful names (eg. alerts-page-up) and using that instead of using the "div" selector or do you have strong feelings towards removing the divs and working on the styles some more?  See updated pull request.